### PR TITLE
도메인 오류 및 테스트 코드 수정

### PR DESCRIPTION
**Pull request by Sene1**
처음으로 해보는거라 잘 될지 모르겠네요

### CODE REVIEW

File: functions/package-school-menu/test/test.ts [hunk 1]
```diff
@@ -20,10 +20,10 @@ describe("[SCHOOL-MENU] Service", function () {
 
     it("fetches menus from NEIS or database properly", (done) => {
         const neisCrawler = new NeisCrawler()
-            .setParameters(SchoolType.HIGH, "K100000460", 2020, 1);
+            .setParameters(SchoolType.HIGH, "N100000191", 2021, 12);
             
         const firestoreAccessor = new MenuDataAccessor(db)
-            .setParameters("K100000460", 2020, 1);
+            .setParameters("N100000191", 2021, 12);
```
Comment (5d-jh) [inline functions/package-school-menu/test/test.ts:26]: 테스트 코드는 최대한 많은 케이스를 커버하는게 좋기 때문에 `it` 함수 호출을 따로 복붙해서 추가하는게 더 좋아요

**Review by 5d-jh — CHANGES_REQUESTED** [review]
로컬에서 테스트 돌려보시고 제대로 작동하는지 확인 부탁드립니다(잘 모르시겠으면 추후에 제가 확인해볼게요)

File: functions/package-school-menu/test/test.ts (hunk 1 above)
Comment (Sene1) [reply functions/package-school-menu/test/test.ts:26]: 혹시 빌드 방법좀 알려주실 수 있나요?
https://app.travis-ci.com/github/5d-jh/school-menu-api/jobs/551467633
오류는 똑같이 발생하는거 같아요

Comment (5d-jh) [reply functions/package-school-menu/test/test.ts:26]: 아 저 ci는 제가 미처 빌드 스크립트 수정을 안해놔서 발생한거네요;; 한번 확인 후 수정해 두겠습니다

Comment (5d-jh) [reply functions/package-school-menu/test/test.ts:26]: 수정해드렸습니다. it 함수 따로 복사 부탁드려요

Comment (5d-jh) [reply functions/package-school-menu/test/test.ts:26]: travis ci에선 schoolinfo.go.kr에 접속을 못해서 자꾸 빌드 오류가 발생합니다. 푸시 후 school menu 관련 테스트만 확인하면 될거에요

Comment (Sene1) [reply functions/package-school-menu/test/test.ts:26]: C:\Users\username\Desktop\school-menu-api-master\functions>yarn workspaces run test
yarn workspaces v1.22.17

> @school-api/common
yarn run v1.22.17
$ mocha dist/test/test.js


  [COMMON] util
    √ throws InvalidResponseBody error when server_message is overriden
    √ creates an object when server_message is not overriden


  2 passing (6ms)

Done in 0.31s.

> @school-api/menu
yarn run v1.22.17
$ mocha dist/test/test.js

TypeError: Cannot read properties of undefined (reading 'INTERNAL')
    at FirebaseNamespace.initializeApp (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\firebase-admin\lib\firebase-namespace.js:387:21)
    at Object.<anonymous> (C:\Users\username\Desktop\school-menu-api-master\functions\package-school-menu\dist\test\test.js:9:53)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at requireOrImport (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\mocha\lib\esm-utils.js:15:12)
    at Object.exports.loadFilesAsync (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\mocha\lib\esm-utils.js:28:26)
    at Mocha.loadFilesAsync (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\mocha\lib\mocha.js:386:19)
    at singleRun (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\mocha\lib\cli\run-helpers.js:149:15)
    at exports.runMocha (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\mocha\lib\cli\run-helpers.js:186:11)
    at Object.exports.handler (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\mocha\lib\cli\run.js:319:11)
    at C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\yargs\lib\command.js:241:49
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed.
Exit code: 1
Command: C:\Program Files\nodejs\node.exe
Arguments: C:\Users\username\AppData\Roaming\npm\node_modules\yarn\lib\cli.js run test
Directory: C:\Users\username\Desktop\school-menu-api-master\functions\package-school-menu
Output:

info Visit https://yarnpkg.com/en/docs/cli/workspaces for documentation about this command.

C:\Users\username\Desktop\school-menu-api-master\functions>npm run dev
npm ERR! Missing script: "dev"
npm ERR!
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\username\AppData\Local\npm-cache\_logs\2021-12-16T01_41_58_809Z-debug.log

C:\Users\username\Desktop\school-menu-api-master\functions>yarn dev
yarn run v1.22.17
error Command "dev" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

C:\Users\username\Desktop\school-menu-api-master\functions>yarn workspaces run test
yarn workspaces v1.22.17

> @school-api/common
yarn run v1.22.17
$ mocha dist/test/test.js


  [COMMON] util
    √ throws InvalidResponseBody error when server_message is overriden
    √ creates an object when server_message is not overriden


  2 passing (6ms)

Done in 0.31s.

> @school-api/menu
yarn run v1.22.17
$ mocha dist/test/test.js

TypeError: Cannot read properties of undefined (reading 'INTERNAL')
    at FirebaseNamespace.initializeApp (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\firebase-admin\lib\firebase-namespace.js:387:21)
    at Object.<anonymous> (C:\Users\username\Desktop\school-menu-api-master\functions\package-school-menu\dist\test\test.js:9:53)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at requireOrImport (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\mocha\lib\esm-utils.js:15:12)
    at Object.exports.loadFilesAsync (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\mocha\lib\esm-utils.js:28:26)
    at Mocha.loadFilesAsync (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\mocha\lib\mocha.js:386:19)
    at singleRun (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\mocha\lib\cli\run-helpers.js:149:15)
    at exports.runMocha (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\mocha\lib\cli\run-helpers.js:186:11)
    at Object.exports.handler (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\mocha\lib\cli\run.js:319:11)
    at C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\yargs\lib\command.js:241:49
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed.
Exit code: 1
Command: C:\Program Files\nodejs\node.exe
Arguments: C:\Users\username\AppData\Roaming\npm\node_modules\yarn\lib\cli.js run test
Directory: C:\Users\username\Desktop\school-menu-api-master\functions\package-school-menu
Output:

info Visit https://yarnpkg.com/en/docs/cli/workspaces for documentation about this command.

yarn run build 사용하면 빌드는 되는데, yarn workspaces run test 사용해서 테스트를 하려고 하면 이렇게 되네요

Comment (5d-jh) [reply functions/package-school-menu/test/test.ts:26]: 음 그럼 일단 수정만 하시고 푸시해주세요

Comment (Sene1) [reply functions/package-school-menu/test/test.ts:26]: > @school-api/common@1.0.0 test
> mocha dist/test/test.js



  [COMMON] util
    √ throws InvalidResponseBody error when server_message is overriden
    √ creates an object when server_message is not overriden


  2 passing (9ms)


> @school-api/menu@1.0.0 test
> mocha dist/test/test.js

Database URL: http://localhost:8080


  [SCHOOL-MENU] Service
    1) fetches menus from NEIS or database properly


  0 passing (2s)
  1 failing

  1) [SCHOOL-MENU] Service
       fetches menus from NEIS or database properly:
     Uncaught Error: Unable to detect a Project Id in the current environment.
To learn more about authentication and Google APIs, visit:
https://cloud.google.com/docs/authentication/getting-started
      at C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\google-auth-library\build\src\auth\googleauth.js:87:31
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
  Caused by: Error
      at Query._get (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\@google-cloud\firestore\build\src\reference.js:1449:23)
      at Query.get (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\@google-cloud\firestore\build\src\reference.js:1438:21)
      at MenuDataAccessor.<anonymous> (dist\src\data\MenuDataAccessor.js:23:18)
      at Generator.next (<anonymous>)
      at C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\tslib\tslib.js:113:75
      at new Promise (<anonymous>)
      at Object.__awaiter (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\tslib\tslib.js:109:16)
      at MenuDataAccessor.get (dist\src\data\MenuDataAccessor.js:18:24)
      at SchoolMenuService.<anonymous> (dist\src\service\SchoolMenuService.js:17:52)
      at Generator.next (<anonymous>)
      at C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\tslib\tslib.js:113:75
      at new Promise (<anonymous>)
      at Object.__awaiter (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\tslib\tslib.js:109:16)
      at SchoolMenuService.getSchoolMenu (dist\src\service\SchoolMenuService.js:16:24)
      at Context.<anonymous> (dist\test\test.js:22:27)
      at processImmediate (node:internal/timers:464:21)



npm ERR! Lifecycle script `test` failed with error:
npm ERR! Error: command failed
npm ERR!   in workspace: @school-api/menu@1.0.0
npm ERR!   at location: C:\Users\username\Desktop\school-menu-api-master\functions\package-school-menu
[..................] / : ERR!   at location: C:\Users\username\Desktop\school-menu-api-master\functions\package-school-menu
> @school-api/info@1.0.0 test
> mocha dist/test/test.js

[..................] / : ERR!   at location: C:\Users\username\Desktop\school-menu-api-master\functions\package-school-menu

  [SCHOOL-INFO] School info parser
(node:13888) Warning: Setting the NODE_TLS_REJECT_UNAUTHORIZED environment variable to '0' makes TLS connections and HTTPS requests insecure by disabling certificate verification.
(Use `node --trace-warnings ...` to show where the warning was created)
    √ parses text from school info (2129ms) C:\Users\username\Desktop\school-menu-api-master\functions\package-school-menu

  [SCHOOL-INFO] School info service
    1) saves keywords or datas  at location: C:\Users\username\Desktop\school-menu-api-master\functions\package-school-menu


  1 passing (5s)
  1 failing

  1) [SCHOOL-INFO] School info service
       saves keywords or datas:
     Uncaught Error: Unable to detect a Project Id in the current environment.
To learn more about authentication and Google APIs, visit:
https://cloud.google.com/docs/authentication/getting-started
      at C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\google-auth-library\build\src\auth\googleauth.js:87:31
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
  Caused by: Error
      at Query._get (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\@google-cloud\firestore\build\src\reference.js:1449:23)
      at Query.get (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\@google-cloud\firestore\build\src\reference.js:1438:21)
      at SchoolInfoDataAccessor.<anonymous> (dist\src\data\SchoolInfoDataAccessor.js:28:18)
      at Generator.next (<anonymous>)
      at C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\tslib\tslib.js:113:75
      at new Promise (<anonymous>)
      at Object.__awaiter (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\tslib\tslib.js:109:16)
      at SchoolInfoDataAccessor.getByKeyword (dist\src\data\SchoolInfoDataAccessor.js:25:24)
      at SchoolInfoService.<anonymous> (dist\src\service\SchoolInfoService.js:12:50)
      at Generator.next (<anonymous>)
      at C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\tslib\tslib.js:113:75
      at new Promise (<anonymous>)
      at Object.__awaiter (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\tslib\tslib.js:109:16)
      at SchoolInfoService.getSchoolInfos (dist\src\service\SchoolInfoService.js:11:24)
      at Context.<anonymous> (dist\test\test.js:31:27)
      at processImmediate (node:internal/timers:464:21)



npm ERR! Lifecycle script `test` failed with error:
npm ERR! Error: command failed
npm ERR!   in workspace: @school-api/info@1.0.0
npm ERR!   at location: C:\Users\username\Desktop\school-menu-api-master\functions\package-school-info

> @school-api/function@1.0.0 test
> mocha dist/test/test.js --exit

[..................] / : ERR!   at location: C:\Users\username\Desktop\school-menu-api-master\functions\package-school-info

  [Integration] school-menu
    1) Issue #41...] / : ERR!   at location: C:\Users\username\Desktop\school-menu-api-master\functions\package-school-info


  0 passing (2s)
  1 failing

  1) [Integration] school-menu
       Issue #41:
     Error: expected 200 "OK", got 500 "Internal Server Error"
      at Suite.<anonymous> (dist\test\test.js:15:14)
      at Generator.next (<anonymous>)
      at C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\tslib\tslib.js:113:75
      at new Promise (<anonymous>)
      at Object.__awaiter (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\tslib\tslib.js:109:16)
      at Context.<anonymous> (dist\test\test.js:13:35)
      at processImmediate (node:internal/timers:464:21)
  ----
      at Test._assertStatus (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\supertest\lib\test.js:296:12)
      at C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\supertest\lib\test.js:80:15
      at Test._assertFunction (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\supertest\lib\test.js:311:11)
      at Test.assert (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\supertest\lib\test.js:201:21)
      at Server.localAssert (C:\Users\username\Desktop\school-menu-api-master\functions\node_modules\supertest\lib\test.js:159:12)
      at Object.onceWrapper (node:events:509:28)
      at Server.emit (node:events:390:28)
      at emitCloseNT (node:net:1681:8)
      at processTicksAndRejections (node:internal/process/task_queues:82:21)



npm ERR! Lifecycle script `test` failed with error:
npm ERR! Error: command failed
npm ERR!   in workspace: @school-api/function@1.0.0
npm ERR!   at location: C:\Users\username\Desktop\school-menu-api-master\functions\package-function

npm run emulator-firestore 명령 한 후에 npm run test --workspaces 하면 이렇게 오류 나네요
아니면 테스트 코드 수정하고 푸시 해드릴까요?

**Comment by Sene1** [comment]
테스트 코드 수정 2

**Comment by 5d-jh** [comment]
배포했습니다. 제가 했을땐 잘 되는 것 같은데 혹시 모르니 테스트 부탁드려요

**Comment by Sene1** [comment]
잘불러와지네요 👍👍

**Comment by 5d-jh** [comment]
넵 기여해주셔서 감사합니다!

**Pull request merged**